shared/actions/actionTypes.js

```diff
@@ -0,0 +1 @@
+export const TBD = 'TBD';
```

Comment:
I think we should set name of function(variable) to more readable.
(can't catch function of TBD at once)